drivers/input/dinDebounce.ts

```diff
@@ -0,0 +1,52 @@
+
+/*
+ * A module to debounce digital input
+ *
+ * Written in TypeScript. Use tsc to convert it to js if needed.
```

Comment:
my 2 cents: for wider compatibility, and ease of use, why not just JS?

Reply:
It's so easy to get the corresponding js code:

> npm install -g typescript // installs typescript

> tsc dinDebounce.ts